Bottom: 3174dfced34e2f2e6b8e33b5a55557a1a99a27a3
Top:    daa7259cc6725c9d36363e06630ace10bf733c3a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 11:01:03 -0500

Refresh of add-host-builder

---
diff --git a/ansible-playbook/staging b/ansible-playbook/staging
index 8058586d..673cb4dd 100644
--- a/ansible-playbook/staging
+++ b/ansible-playbook/staging
@@ -1,4 +1,4 @@
-builder.novatech-llc.com ansible_host=127.0.0.1 ansible_port=20022 ansible_user='vagrant' ansible_ssh_private_key_file='/home/coopera/Developer/testdaedalus/.vagrant/machines/bbworker/virtualbox/private_key'
+builder.novatech-llc.com ansible_host=172.16.64.4 ansible_user=ansibleremote
 
 [bacula]
 testdaedalus.novatech-llc.com
